kinto_attachment/__init__.py

```diff
@@ -69,6 +69,8 @@ def includeme(config):
     # Enable attachment backend.
     if 'storage.base_path' in storage_settings:
         config.include('pyramid_storage.local')
+    elif 'storage.gcloud.credentials' in storage_settings:
+        config.include('pyramid_storage.gcloud')
     else:
         config.include('pyramid_storage.s3')
```

Comment:
Should we emit some sort of warning or error if a user has both `gcloud` and `s3`-looking credentials enabled in the config file? Say they were using glcoud, switched to s3, but forgot to remove the `storage.gcloud.credentials` option.

Reply:
We could. 

It wasn't the case for the `base_path` setting, so I thought it was acceptable for gcloud and s3.